Society for Neuroscience Abstract 2016.js

```diff
@@ -0,0 +1,446 @@
+{
+	"translatorID": "1ed36c21-de68-4ba4-a27f-d724ce258db4",
+	"label": "Society for Neuroscience Abstract 2016",
+	"creator": "Kouchi C. Nakamura, PhD; kouichi.c.nakamura@gmail.com",
+	"target": "^https?://www\\.abstractsonline\\.com/pp8/#!/(4036|4376)/presentation/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2017-09-28 06:13:46"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2017 Kouichi C. Nakamura, PhD
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+/*
+This Zotero translator works with Soceity for Neuroscience Abstract from 2016 onwards
+
+**Caveats**
+Does not support "mulitple" yet
+Does not support taking a snapshot for "attachments"
+The dynamically generated HTML of abstractsonline.com does not allow above operations
+
+supported years: four digits after /pp8/ in the url specifies a year
+4376 ... 2017
+4071 ... 2016
+
+Ideally "target" should be as below to support "multiple" entries
+^https?://www\.abstractsonline\.com/pp8/#!/\d+/(presentation|presentations|sessions|participants)/
+
+If you find a failure or error, please let me know:
+Dr Kouichi C. Nakamura
+kouichi.c.nakamura@gmail.com
+*/
+
+
+function detectWeb(doc, url) {
+
+	if (url.search(/\/presentation\/\d+/) != -1) {
+		return 'conferencePaper';
```

Comment:
This is actually a tricky bit. 

I personally wanted it to be even `journalArticle`, because it will be easier to cite in papers. `conferencePaper` does not have a field for the abbreviated journal title, so you will end up editing citation to include `Soc. Neurosci. Abstr.` etc manually. 

But I thought, because it is not a journal per se, many people find it weird. 

The downside of using `presentation` is that because it does not have `pages` field, it's hard to accommodate presentation id. This is related to why presentation id is stored in `pages` field.